app/routes.py

```diff
@@ -148,9 +148,20 @@ def unfollow_user(user_id, id):
 # get a list of users that current user is following
 @app.route('/api/v1/users/<id>/following', methods=['GET'])
 def get_following(id):
-    user = User.query.filter_by(id=id).first_or_404()
-    response = "something"
-    return jsonify(response)
+    current_user = User.query.filter_by(id=id).first_or_404()
+    user_obs = current_user.followed.all()
+    users = []
+    for user in user_obs:
+        response = {
```

Comment:
Nice reuse of methods! I think this will work perfectly for our needs. Maybe a future refactor would include pulling this logic into a helper method.